x/protorev/types/genesis.go

```diff
@@ -19,7 +19,13 @@ var MaxInputAmount = sdk.NewInt(16_384)
 var StepSize = sdk.NewInt(1_000_000)
 
 // Max iterations for binary search (log2(16_384) = 14)
-const MaxIterations = 14
+const MaxIterations int = 14
+
+// Max number of routes that can be arbitraged per tx (default of 6)
```

Comment:
I'm trying to understand this, does this mean 6 routes each containing 3 pool routes? So it'd look something like [[route1][route2][route3]],  [[route1a][route2a][route3a]], [[route1b][route2b][route3b]] ... 6times in genesis

Reply:
Yes exactly. But this is also dependent on the route that is taken. Routes that include a stable swap pool inheritely are more expensive to backrun. Because of this we count routes that include a stable swap pool with more weight. Currently the distribution is 2:5 for balancer : stable weights. The basic idea here is that `MaxIterableRoutesPerTx` and `MaxIterableRoutesPerBlock` closely follow the actual ms that we expect the module to take and route weights are the approximate amount of time it takes to actually traverse the route (in ms).